api/v1/views/places.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+"""DOCSTRING FOR MODULE"""
+from api.v1.views import app_views
+from flask import abort, jsonify, request
+from models import storage
+from models.place import Place
+
+
+@app_views.route(
+        '/cities/<city_id>/places', methods=['GET'], strict_slashes=False)
+def show_places(city_id):
+    """retrieves all Place objects"""
+    city = storage.get('City', city_id)
+    places_list = []
+    if city is not None:
+        for place in city.places:
+            places_list.append(place.to_dict())
+        return jsonify(places_list)
```

Comment:
Maybe consider adding the `200` response code in a successful return for clarity.